doc/man7/proxy-certificates.pod

```diff
@@ -0,0 +1,359 @@
+=pod
+
+=head1 NAME
+
+proxy-certificates - Proxy certificates in OpenSSL
+
+=head1 DESCRIPTION
+
+Proxy certificates are defined in
+L<RFC 3820|https://tools.ietf.org/html/rfc3820>.  They are used to
+extend rights to some other entity (a computer process, typically, or
+sometimes to the user itself).  This allows the entity to perform
+operations on behalf of the owner of the EE certificate.
+
+The requirements for a valid proxy certificate are:
+
+=over 4
+
+=item *
+
+They are issued by an End Entity, either a normal EE certificate, or
+another proxy certificate.
+
+=item *
+
+They must not have the extensions B<subjectAltName> or
+B<issuerAltName>.
+
+=item *
+
+They must have the extension B<proxyCertInfo>.
```

Comment:
Same here.

Reply:
Must have the proxyCertInfo extension.

Reply:
> They must not have the extensions B\<subjectAltName\> or
B\<issuerAltName\>.

In German language this would be the correct ordering, too, so I can understand your reluctance. I still believe that in English it's other way around , but I won't insist on it. I'll just content myself with the fact that two native speakers voted in favour of my suggestion ;-)

Reply:
@richsalz speaks American, I speak Australian.
Neither is quite English.

Reply:
It surprises me that you consider American and Australian English not to be proper English ;-)

Reply:
I doubt that this issue depends on any geographical variant of Standard English.  From having looked around, I have seen no trace of British vs TheRestOfTheAnglosphere dispute on this subject, so...

Reply:
I deliberately avoided the word _proper_ :smiley: